src/libc_installation.zig

```diff
@@ -524,7 +524,7 @@ pub const CCPrintFileNameOptions = struct {
 };
 
 /// caller owns returned memory
-fn ccPrintFileName(args: CCPrintFileNameOptions) ![:0]u8 {
+pub fn ccPrintFileName(args: CCPrintFileNameOptions) ![:0]u8 {
```

Comment:
As far as I can tell this did not end up getting called from outside this file, right?